docs/rst/enhancements/easy_mode/easy_mode.rst

```diff
@@ -0,0 +1,138 @@
+.. _easy_mode:
+
+Easy Mode
+=========
+
+The new ``Vulcanexus Easy Mode`` aims to simplify, enhance and optimize the deployment of any ROS 2 application reinforcing the overall out-of-the-box user experience.
+Moreover, it claims to be the default discovery behavior within :ref:`Vulcanexus <vulcanexus_introduction>`.
+This section reveals its significance, operating mode and impact.
+
+Quick Overview
+^^^^^^^^^^^^^^
+
+The ``Vulcanexus Easy Mode`` is a builtin discovery mode that simplifies to the most the deployment of ROS 2 applications with the use of `Discovery Server <https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/discovery_server.html>`_.
+To enable this feature the user only needs to set the environment variable ``EASY_MODE=<ip>``.
+This ``<ip>`` can be either the IP address of the current host or the ``<ip>`` of an external host in the same LAN acting as the central point of discovery (``master``).
+A new discovery server will be spawned in background in the domain specified by the ``ROS_DOMAIN_ID`` environment variable.
+The following diagram illustrates the concept:
+
+.. image:: ../../figures/enhancements/easy_mode/easy_mode_quick_intro.png
+    :align: center
+    :width: 55%
+
+The advantages of the ``Vulcanexus Easy Mode`` can be explored in the next section.
+
+.. _easy_mode_benefits:
+
+Benefits
+^^^^^^^^
+
+The use of ``Vulcanexus Easy Mode`` brings the following benefits:
+
+* **Simplicity**: The user does not need to worry about configuring the middleware, enhancing the out-of-the-box experience.
+  The user easily manages connections between domains and hosts only when needed.
+* **Scalability**: The number of discovery packets is drastically reduced, and the system scalability improves.
+* **Reliability**: The new discovery mode is more reliable than Simple discovery with multicast, especially in WiFi scenarios, as it uses peer to peer TCP.
+
+Feature Insights
+^^^^^^^^^^^^^^^^
+
+This sub-section provides a detailed explanation of the ``Vulcanexus Easy Mode``.
+
+Motivation
+----------
+
+ROS 2 users often expect ROS 2 applications to run out-of-the-box without caring about middleware configuration or network setup.
+As a result, experience has revealed that the default `DDS Simple discovery mechanism <https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/simple.html#simple-discovery-settings>`_ with multicast is not always the best choice for the majority of use cases.
+However, `eProsima's Discovery Server <https://fast-dds.docs.eprosima.com/en/latest/fastdds/discovery/discovery_server.html>`_ mechanism effectively removes the need for multicast discovery traffic and presents a flexible solution for different network topologies.
+
+.. list-table::
```

Comment:
Move this after the next paragraph to have the explanation first. And I'd remove the domains from here. The first time you mention or explain domain is in the next section so I prefer to leave that for more advanced users